test/.eslintrc

```diff
@@ -8,5 +8,8 @@
   },
   "globals": {
     "_": true,
+  },
+  "rules": {
```

Comment:
What's this new rule for?

Reply:
As mentioned in the commit message, this is so that ESLint displays a warning whenever a `XXX`, `TODO` or `FIXME` comment occurs within the test suite.

Reply:
Ah okay, in that case we've been using `// @TODO` for this purpose but would prefer not to have this show up in the warning, can you convert the comment to @TODO and remove the rule? 

When refactoring we'll go through the @TODO comments anyway

I'll merge after this change, thanks!